### DISCUSSION TITLE
Intial Logo Designs

### DISCUSSION THREAD

**Discussion by Prakash4844** [Prakash4844/tit#20] [Ideas]
![dfas](https://github.com/Prakash4844/tit/assets/81550376/3bb5c74f-9f04-4c57-a3c9-cf8b89774f3d)
![OxIG](https://github.com/Prakash4844/tit/assets/81550376/9919606b-7a57-4596-a491-03012990fab3)
![dsaf](https://github.com/Prakash4844/tit/assets/81550376/b7a24507-3985-43da-b5ee-567b8baa69ec)
![OIG CQ7cud](https://github.com/Prakash4844/tit/assets/81550376/9bc29b75-7e3b-4b8f-ae97-d1bebe8bc74b)
![OIG 6NasdPT_f3](https://github.com/Prakash4844/tit/assets/81550376/4fb5dab6-49a0-4660-b803-75c873c340e4)
![fbfdh](https://github.com/Prakash4844/tit/assets/81550376/01a15f8b-ef07-47d2-9203-b9bebcf18f5b)
![gfOIG](https://github.com/Prakash4844/tit/assets/81550376/75e8f361-370e-4bdc-b9a1-b9d4e57df46f)
![OIG](https://github.com/Prakash4844/tit/assets/81550376/e3de9f06-8ef6-4930-b227-a1bcf781b9ef)
![OIG 76Ai7upIKON2kfa1bQ](https://github.com/Prakash4844/tit/assets/81550376/b4af2175-07cb-42b4-bec1-a40f88a70155)

These are some of the prototype logos. which one looks the best?

**Comment by Prakash4844** [comment]
@sagargoswami2001 @SakuraBlossomTree What do you think?

**Reply by SakuraBlossomTree** [reply]
I think the second one is good 

It looks simple for me

**Reply by Prakash4844** [reply]
I also liked that one.

**Comment by Prakash4844** [comment]
I have Uploaded the logo in repo and updated Readme Accordingly.
